Cargo.toml

```diff
@@ -10,30 +17,30 @@ rand = "0.8"
 unroll = "0.1"
 num_cpus = "1"
 rayon = "1"
-smallvec = { version = "*", features = ["const_generics", "const_new", "serde"] }
-crossbeam = "*"
-sha3 = { git = "https://github.com/RustCrypto/hashes.git", rev = "7a187e934c1f6c68e4b4e5cf37541b7a0d64d303" }
-lazy_static = "*"
+smallvec = { version = "1.13", features = ["const_generics", "const_new", "serde"] }
+crossbeam = "0.8.4"
+sha3 = { package = "sha3_ce", version = "=0.10.6" }
```

Comment:
Is that related to conflicting requirements from zk_evm packages eventually?

Reply:
Yes; republishing was the easiest option.